Framework/Algorithms/src/ConvertUnits.cpp

```diff
@@ -465,21 +464,18 @@ void ConvertUnits::convertViaTOF(Kernel::Unit_const_sptr fromUnit,
 
       /// @todo Don't yet consider hold-off (delta)
       const double delta = 0.0;
-      std::vector<double> vals(outputWS->x(i).begin(), outputWS->x(i).end());
-      // Convert the input unit to time-of-flight
-      localFromUnit->toTOF(vals, emptyVec, l1, l2, twoTheta, emode, efixed,
-                           delta);
 
+      // TODO toTOF and fromTOF need to be reimplemented outside of kernel
```

Comment:
Are you keeping track of these `TODOs` somewhere? Otherwise I would recommend to choose a more unique label, something like `TODOHISTOGRAM`.